NEWS.md

```diff
@@ -1,73 +1,60 @@
-3.22.1
+3.23.1
 ======
 
-## Highlights
 
- * Sending SNMP traps: Using the new `snmp()` destination, incoming log messages
-   can be converted to SNMP traps, as the fields of the SNMP messages can be
-   customized with macros. (#2693)
+## Features
 
- * `$(template)` dynamic binding: Extends the $(template) template function to
-   allow dynamic binding. For example, the name of the template to be invoked
-   can come from the message (name-value pairs). (#2716)
+ * The `redis()` destination now handles any number of command parameters.
+   (#2816)
 
- * `syslog()`, `network()`: Add `dynamic-window-size()` option to enable dynamic
-   flow control that distributes the specified amount of window between active
-   connections at runtime. This can be used in low-memory environments, where
-   only a small subset of the active clients sends messages at high rate.
-   (#2772)
+ * The `format()` option of `date-parser()` supports lists. From now on, a
+   single `date-parser()` instance is able to process different date formats,
+   making it easy to catch on when some programs change the way they log
+   information. (#2779)
 
-## Features
+ * Add relocation support for `disk-buffer()`. The `relocate` subcommand of
+   `dqtool` can be used to move a single or multiple queue files. (#2855)
 
- * `match()`: Add support for the `template()` option (#2715)
- * `add-contextual-data()`: Allow using templates in name-value pairs (#2711)
- * Add support for floating point operations in template functions (#2742)
- * Add support for usec precision when parsing time (#2709)
+ * `file(), pipe()`: The `time-reap()` option now can be set or disabled for
```

Comment:
I would also notify the users, that for non-templated sources, time-reap is now disabled by default.

Reply:
It has a separate entry in `Other changes`.

Reply:
Oh my bad! Thanks! I also meant destination, don't mind me :D